service/controller/app/v1/resource/chart/resource.go

```diff
@@ -11,19 +11,24 @@ import (
 
 const (
 	// Name is the identifier of the resource.
-	Name = "chartv1"
+	chartAPIVersion           = "application.giantswarm.io"
```

Comment:
Godoc is wrong when this is added. Lets have `Name` first and a newline to separate the un-exported constants.